python-type-hints-multiple-types/06_type_aliases.py

```diff
@@ -0,0 +1,14 @@
+from typing import TypeAlias
+
+
+EmailComponents = tuple[str, str] | None
```

Comment:
Do we need to include the old-style way of defining an alias? Maybe only the new one will suffice?

```python
EmailComponents: TypeAlias = tuple[str, str] | None
```